modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/ratings.js

```diff
@@ -12,734 +12,9 @@
-							ratingThumbUpCssClassOn = ratingThumbUp.hasClass(
-								cssClassesOn
-							);
-						}
-
-						var thumbDownMessage = '';
-						var thumbUpMessage = '';
-
-						if (ratingThumbDown) {
-							if (ratingThumbDownCssClassOn) {
-								thumbDownMessage = Liferay.Language.get(
-									'you-have-rated-this-as-bad'
-								);
-							}
-							else {
-								thumbDownMessage = Liferay.Language.get(
-									'rate-this-as-bad'
-								);
-							}
-
-							ratingThumbDown.attr('title', thumbDownMessage);
-
-							ratingThumbDown
-								.one('.votes')
-								.html(thumbScore.negativeVotes);
-						}
-
-						if (ratingThumbDown && ratingThumbUpCssClassOn) {
-							thumbUpMessage = Liferay.Language.get(
-								'you-have-rated-this-as-good'
-							);
-						}
-						else if (
-							ratingThumbDown &&
-							!ratingThumbUpCssClassOn
-						) {
-							thumbUpMessage = Liferay.Language.get(
-								'rate-this-as-good'
-							);
-						}
-						else if (
-							!ratingThumbDown &&
-							ratingThumbUpCssClassOn
-						) {
-							thumbUpMessage = Liferay.Language.get(
-								'unlike-this'
-							);
-						}
-						else if (
-							!ratingThumbDown &&
-							!ratingThumbUpCssClassOn
-						) {
-							thumbUpMessage = Liferay.Language.get('like-this');
-						}
-
-						if (ratingThumbUp) {
-							ratingThumbUp.attr('title', thumbUpMessage);
-
-							ratingThumbUp
-								.one('.votes')
-								.html(thumbScore.positiveVotes);
-						}
-					}
-				},
-			},
-		});
-
-		var LikeRatingImpl = A.Component.create({
-			EXTENDS: A.ThumbRating,
-
-			NAME: 'LikeRatingImpl',
-
-			prototype: {
-				renderUI() {
-					var instance = this;
-
-					var cssClasses = instance.get('cssClasses');
-
-					A.ThumbRating.superclass.renderUI.apply(this, arguments);
-
-					var elements = instance.get('elements');
-
-					elements.addClass(cssClasses.off);
-					elements.item(0).addClass(cssClasses.up);
-				},
-			},
-		});
-
-		var LikeRating = A.Component.create({
-			EXTENDS: ThumbRating,
-
-			NAME: 'LikeRating',
-
-			prototype: {
-				_createRating() {
-					var instance = this;
-
-					var namespace = instance.get(STR_NAMESPACE);
-
-					instance.ratings = new LikeRatingImpl({
-						boundingBox: '#' + namespace + 'ratingLike',
-						cssClasses: {
-							down: '',
-							element: 'rating-off',
-							hover: 'rating-on',
-							off: 'rating-off',
-							on: 'rating-on',
-							up: '',
-						},
-						srcNode: '#' + namespace + 'ratingLikeContent',
-					}).render();
-				},
-
-				_getThumbScores(entries) {
-					return {
-						positiveVotes: entries,
-					};
-				},
-			},
-		});
-
-		Ratings.LikeRating = LikeRating;
-		Ratings.StarRating = StarRating;
-		Ratings.ThumbRating = ThumbRating;
-
-		Liferay.Ratings = Ratings;
-	},
-	'',
-	{
-		requires: ['aui-rating'],
-	}
-);
+/**
+ * @deprecated As of Athanasius (7.3.x), with no direct replacement
+ */
+AUI.add('liferay-ratings', () => {}, '', {
```

Comment:
Just the deprecation notice is enough. Nuking the module is not necessary!

Reply:
Ok!